test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -177,6 +177,13 @@ export default class IdentityPageObject extends BasePageObject {
     return this.form.elementExist(selector);
   }
 
+  identifierFieldExistsForPIVView() {
+    if (userVariables.v3) {
+      return this.identifierFieldExists('[data-se="identifier"]');
```

Comment:
Does it make sense to use a `getByRole` with the field label as a parameter instead?